.travis.yml

```diff
@@ -3,8 +3,6 @@ language: php
 sudo: false
 
 php:
-  - 5.6
```

Comment:
@raftaar1191 this is commit related to this issue?

Reply:
Nope I am justing removing that only